Source/WebCore/css/calc/CSSCalcTree+Parser.cpp

```diff
@@ -1459,14 +1459,15 @@ std::optional<TypedChild> parseCalcSum(CSSParserTokenRange& tokens, int depth, P
     if (checkDepth(depth) != ParseStatus::Ok)
         return std::nullopt;
 
+    auto originalTokens = tokens.span();
```

Comment:
This is fine. 

To avoid needing to keep this range in sync with other changes, we could consider tracking a "previousTokenWasWhitespace" bool instead. That would also make the loop more optimal.

I thought about commenting this before, and then didn't think it mattered much, and then we crashed, so I guess I'll mention it now, in case it sounds appealing.